### PR TITLE
New Template: cisco_nxos_show_license_usage

### PR DESCRIPTION
**Pull request by network-dave**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

Cisco NXOS - show license usage

##### SUMMARY

N/A

### CODE REVIEW

File: templates/cisco_nxos_show_license_usage.textfsm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Value FEATURE (\S+)
+Value INSTALLED (Yes|No)
+Value LICENSE_COUNT (\d+|-)
+Value STATUS (Unused|In\s+use)
+Value EXPIRY_DATE (\S+|\s*?)
+Value COMMENTS (.+)
+
+Start
+  ^\s+Count\s*$$ -> Begin
```
Comment (FragmentedPacket) [inline templates/cisco_nxos_show_license_usage.textfsm:9]:
```suggestion
Start
  ^Feature\s+Ins\s+Lic\s+Status\s+Expiry\s+Date\s+Comments - > Begin
  ^\s*$$
  ^. -> Error

Begin
  ^\s+Count

Comment (network-dave) [reply templates/cisco_nxos_show_license_usage.textfsm:9]: Done

Comment (FragmentedPacket) [reply templates/cisco_nxos_show_license_usage.textfsm:9]: I still see the same thing is was prior.

Comment (network-dave) [reply templates/cisco_nxos_show_license_usage.textfsm:9]: The code has been updated like you suggested though, the update is visible in my forked repo. Is there anything else I need to do ?

Comment (FragmentedPacket) [reply templates/cisco_nxos_show_license_usage.textfsm:9]: You need to commit and push it up to your branch on Github for it to update here.

Comment (network-dave) [reply templates/cisco_nxos_show_license_usage.textfsm]: Woops, looks like I forgot to push on the branch indeed. Just fixed that.

**Pull request merged**